package.json

```
@@ -28,6 +28,17 @@
     "emoji-to-name.json",
     "name-to-emoji.json"
   ],
+  "jsdelivr": [
+    "index.json",
+    "name-to-emoji.json",
+    "emoji-to-name.json"
+  ],
+  "browser" : [
+    "index.json",
+    "name-to-emoji.json",
+    "emoji-to-name.json"
+  ],
+  "main": "index.json",
```

```suggestion
  "main": "index.json",
```

Thank you for working on this! From the number of times you tried you seem a bit new here on GH, so I’d just like to say how awesome it is that you contribute! 👏

I do propose only adding the `main` field, as that works in all places. And should solve GH-9 fine?
You can click “commit suggestion” above to apply my suggestion!

---

Hey,
Thank you very much. Really appreciate the help, and sorry for not being good at this.